.buildkite/notebook-parameters.yml

```
@@ -4,4 +4,4 @@ epochs: 2
 walk_length: 5
 batch_size: 5
 n_estimators: 2
-n_predictors: 2
+n_predictors: 1
```

I don't think this is the best unless it's a huge speed-up. My thinking is that there's more likely to be a difference between 1 and 2, than between 2 and 3, or 2 and 4. E.g. axis reductions might not happen correctly, or there might be special cases for a single predictor.

That is,

```suggestion
n_predictors: 2
```

What do you think?